iroh-net/src/netcheck.rs

```diff
@@ -949,6 +949,11 @@ mod tests {
         // create raw ICMP pings and we'll have to silenty accept this test is useless (if
         // we could, this would be a skip instead).
         let have_pinger = Pinger::new().is_ok();
+        if !have_pinger {
+            error!("pinger disabled, test effectively skipped");
+        } else {
+            info!("pinger enabled");
```

Comment:
while useful this will pop up to the user, maybe both should be debug

Reply:
This is in a test.  I very much want to see this.

Reply:
yeah in a test it makes sense